lib/mix/lib/mix/config.ex

```diff
@@ -5,6 +5,9 @@ defmodule Mix.Config do
   @moduledoc ~S"""
   A simple configuration API and functions for managing config files.
 
+  This module is deprecated, use `Elixir.Config` and `Elixir.Config.Reader`
+  instead.
```

Comment:
```suggestion
  This module is deprecated, use the modules `Config` and `Config.Reader`
  from Elixir's standard library instead.
```

WDYT?

Reply:
Perfect, keeps it better in the normal style! Thanks!